app/assets/stylesheets/components/_slider.scss

```diff
@@ -0,0 +1,88 @@
+$track-w: 16em;
+$track-h: .625em;
+$track-bg: $grey-4;
+$track-fill: linear-gradient(90deg, $grey-2 25%, $grey-2 100%);
+
+$thumb-d: 1.5em;
```

Comment:
What does `-d` mean? Can we use more expressive variable names like `$track-width`, `$track-height` etc?